core/__tests__/modules/configWriter.ts

```diff
@@ -68,13 +68,13 @@ describe("modules/configWriter", () => {
       await App.destroy({ truncate: true });
     });
 
-    test.skip("provides a url-friendly path from the name in the object", async () => {
```

Comment:
Are there any more skipped tests?

Reply:
Nope! Did a global search and seems like we're in good shape.